README.md

```diff
@@ -124,6 +129,36 @@ If an argument other than an ArgMatcher (like `any`, `anyNamed()`, `argThat`,
 used for argument matching. If you need more strict matching consider use
 `argThat(identical(arg))`.
 
+Also note that `null` cannot be used as an argument adjascent to ArgMatcher
+arguments, nor as an un-wrapped value passed as a named argument. For example:
+
+```dart
+verify(cat.hunt("back yard", null)); // OK: no ArgMatchers.
+verify(cat.hunt(argThat(contains("yard")), null)); // BAD: adjascent null.
+verify(cat.hunt(argThat(contains("yard")), argThat(isNull))); // OK: wrapped in ArgMatcher.
+verify(cat.eatFood("Milk", hungry: null)); // BAD: null as named argument.
+verify(cat.eatFood("Milk", hungry: argThat(isNull))); // BAD: null as named argument.
+```
+
+## Named arguments
+
+Mockito currently has an awkward nuisance to it's syntax: named arguments and
+argument matchers require more specification than you might think. Namely, you
```

Comment:
I'm wondering if we can do the 1000 foot explaination on this. Something like: This is because the language doesn't provide the name of the argument in the invocation and we can't rely on the position of a named argument.

Reply:
Good call. Done.